src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/MobEventsUtils.java

```diff
@@ -0,0 +1,221 @@
+package com.minecolonies.coremod.entity.ai.mobs.util;
+
+import com.minecolonies.api.configuration.Configurations;
+import com.minecolonies.api.util.LanguageHandler;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.colony.Colony;
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.ColonyView;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utils for Colony mob events
+ */
+public class MobEventsUtils
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1118)